env.js

```diff
@@ -169,14 +169,6 @@ function config ( ) {
   env.pushover_api_token = readENV('PUSHOVER_API_TOKEN');
   env.pushover_user_key = readENV('PUSHOVER_USER_KEY') || readENV('PUSHOVER_GROUP_KEY');
 
-  // TODO: clean up a bit
```

Comment:
I agree, but I removed this line as it overwrites the environment variables with custom variables as defined in the `database_configuration.json`. This is confusing when you assume that you are connecting to the Mongo-database by the credentials in the environment, when you in fact are using the credentials from the file.